lib/tasks/heart_rate_data.rake

```diff
@@ -0,0 +1,8 @@
+namespace :heart_rate_data do
+  desc "Update heart rate data for each user for Fitbit"
+  task sync: :environment do
+    HeartRateData.all.each do |heart_rate_data|
+      heart_rate_data.update_heart_rate_data!
+    end
```

Comment:
Sure! A few questions before I start:
1. Why the last user?
2. Currently there is a one-to-many relationship between users and heart_rate_data(s). Shall I keep this? Or would you prefer a one-to-one relationship where a user only has one heart_rate_data entry?

In terms of refreshing the API token, this gets handled by `update_heart_rate_data!` so I don't need to explicitly do anything.